sanic/request.py

```diff
@@ -79,7 +79,7 @@ def token(self):
         :return: token related to request
         """
         auth_header = self.headers.get('Authorization')
-        if 'Token ' in auth_header:
+        if auth_header is not None and 'Token ' in auth_header:
```

Comment:
Would it make more sense to provide the empty string as the default instead of None? `auth_header = self.headers.get('Authorization', '')`. Then `if 'Token ' in auth_header` will fail rather than raise an exception.